config/manager/manager.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
```

Comment:
what is this file for?

Reply:
just make the `kubectl kustomize .` command can work in  `starrocks-kubernetes-operator/config/manager`.

Reply:
ok